runtime/doc/change.txt

```diff
@@ -1599,6 +1598,9 @@ r	Automatically insert the current comment leader after hitting
 o	Automatically insert the current comment leader after hitting 'o' or
 	'O' in Normal mode.  In case comment is unwanted in a specific place
 	use CTRL-U to quickly delete it. |i_CTRL-U|
+							*fo-/*
+	When 'o' is included: do not insert the comment leader for a //
```

Comment:
```suggestion
/	When 'o' is included: do not insert the comment leader for a //
```

Reply:
OK, what is the point of that? I assumed that was a typo and omitted it.

Reply:
This is the name of the option.

Reply:
🤦